### ISSUE TITLE
Why did you not randomly initialize those weights?

### ISSUE THREAD

**Issue by ajesujoba** [RandyZhouRan/MELM#6]
Thanks for the good work. Looking at your code, I discovered that you initialized the embedding of the language tokens and the NER labels that you added to the vocabulary using some existing embeddings in the Roberta model. What is the justification for that?

https://github.com/RandyZhouRan/MELM/blob/662c6dcb7dbed52df45a7036f8cfb14cd8f92a37/trainer.py#L155

**Comment by RandyZhouRan** [comment]
As we explained in the paper, we used embeddings of words sematically related to the class names as initialization (e.g. person for <B-PER>), such that MELM can utilize the pretrained semantic information during training
